packages/gafl-webapp-service/README.md

```diff
@@ -24,7 +24,8 @@ To run from this directory:
 | GOV_PAY_APIKEY             | GOV pay access identifier                       |    no    | Yes                 |                               |
 | GOV_PAY_REQUEST_TIMEOUT_MS | Timeout in milliseconds for API requests        |    no    | Yes                 |                               |
 | FEEDBACK_URI               | Location of feedback survey                     |    no    | #                   |                               |
-
+| ANALYTICS_PRIMARY_PROPERTY | Analytics ID for tracking inc ecommerce         |   yes    |                     |                               |
```

Comment:
You won't be able to do a full run through, as if they're not set then the Hapi Gapi plugin isn't initialised and request.ga will be undefined. I can obviously check that so that it doesn't break, but then errors would end up hidden. Perhaps a log when Hapi Gapi doesn't get initialised, so we know that tracking isn't taking place

Reply:
Good point Phil.   This might be something that we should improve in hapi gapi itself.

Perhaps rather than not initialising the plugin if these aren't set, only add the entries to the propertySettings if the individual variables are set:
```
        propertySettings: [
          {
            id: process.env.ANALYTICS_PRIMARY_PROPERTY,
            hitTypes: ['pageview', 'event', 'ecommerce']
          },
          {
            id: process.env.ANALYTICS_EXGOV_PROPERTY,
            hitTypes: ['pageview']
          }
        ],
```